pymoveit2/moveit2.py

```diff
@@ -1148,6 +1153,59 @@ def allowed_planning_time(self, value: float):
 
         self.__move_action_goal.request.allowed_planning_time = value
 
+    def add_collision_mesh(
+        self,
+        id: str,
+        position: Union[Point, Tuple[float, float, float]],
+        quat_xyzw: Union[Quaternion, Tuple[float, float, float, float]],
```

Comment:
This function does not currently support `Point`/`Quaterion` as indicated with typing.

To support it, the following should to be added.
```python
if isinstance(position, Point):
    pose.position = position
else:
    pose.position.x = float(position[0])
    pose.position.y = float(position[1])
    pose.position.z = float(position[2])
if isinstance(quat_xyzw, Quaternion):
    pose.orientation = quat_xyzw
else:
    pose.orientation.x = float(quat_xyzw[0])
    pose.orientation.y = float(quat_xyzw[1])
    pose.orientation.z = float(quat_xyzw[2])
    pose.orientation.w = float(quat_xyzw[3])
```